source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ACRChatWindow.mm

```diff
@@ -146,8 +144,61 @@ - (void)renderCards:(NSString *)card
                            widthConstraint:adaptiveCardsWidth
                                   delegate:self.adaptiveCardsDelegates];
     }
+    return renderResult.view;
+}
 
-    [adaptiveCardsViews addObject:renderResult.view];
+- (void)renderCards:(NSString *)card
```

Comment:
could this method or the method in line 129 be renamed?  while some rendering is being performed inside of it, it seems like it's adding the rendered card to the cards list and the plural in card would suggest as it is rendering multiple cards; a possible update could be having the method in line 129 changed to: renderCardFromJson and this method changed to renderCard (or it could follow the pattern from lines 59 & 66 where the methods start with the word 'insert')

Reply:
you are correct. But this is an existing function & public. Its function hasn't changed, so I rather keep it, for now, to keep it simple and on point to addressing the issue of supporting dynamic font type.